src/ray/stats/metric_defs.cc

```diff
@@ -16,8 +16,6 @@
 
 #include "ray/util/size_literals.h"
 
-using namespace ray::literals;
```

Comment:
You shouldn't do that, it's used for size literals, just `// NOLINT` to suppress